src/login-store.js

```diff
@@ -5,8 +5,11 @@ import { routerReducer } from "react-router-redux";
 import { reducer as formReducer } from "redux-form";
 import { intlReducer } from "react-intl-redux";
 
+import loginReducer from "login/LoginMain/LoginMain_reducer";
+
 
 const eduIDLoginApp = combineReducers({
+  config: loginReducer,
```

Comment:
With a generic `AppLoader` component I'm expecting this reducer to be something generic like `initApp: appLoader` (maybe remove reducer in the import variable).

**My hope with this generic  `AppLoader` component  (and any generic other component) is:**
- That you'll be able to read this like a book, first loading app, then getting config...whatever...
- That if we can make stuff generic we can see where all of the apps  are the same (i.e. they all need to load) and where they are different